packages/editor/src/panels/assets/resources.tsx

```diff
@@ -303,21 +324,35 @@ function BottomPaginationNavBar({ handleScrollToPage }) {
 
 function ResourceItems() {
   const { t } = useTranslation()
-  const { resources, staticResourcesPagination } = useAssetsQuery()
+  const { resourcesLoading, resources, staticResourcesPagination } = useAssetsQuery()
   const pages = Math.ceil(resources.length / (ASSETS_PAGE_LIMIT + calculateItemsToFetch()))
   const pageRefs = useRef<(HTMLDivElement | null)[]>([]) // Create a ref array
+  const fileIconsLoaded = useHookstate(0)
+  const fileIcosToLoad = useHookstate(0)
 
   const handleScrollToPage = (pageIndex: number) => {
     if (pageRefs.current[pageIndex]) {
       pageRefs.current[pageIndex]!.scrollIntoView({ behavior: 'smooth', block: 'start' })
     }
   }
 
+  const isStillLoadingIcons = () => {
+    return !(fileIconsLoaded != fileIcosToLoad)
+  }
```

Comment:
lets make this a boolean
```suggestion
  const isStillLoadingIcons = fileIconsLoaded.value !== fileIconsToLoad.value
```